features/steps/formatting_steps.rb

```diff
@@ -251,7 +251,7 @@
 end
 
 Then(/^I should see a red error message$/) do
-  run_output.should include(red("⌦  " + SAMPLE_PODS_ERROR.gsub('error: ', '')))
+  run_output.should include(red("❌  " + SAMPLE_PODS_ERROR.gsub('error: ', '')))
```

Comment:
Consistency wise - wouldn't it be better if we used the one from line 238 `red("✗")` ?